### ISSUE TITLE
REST API: catalogProductAttributeManagementV1 Get returns Array insted of Boolean for is_default

### ISSUE THREAD

**Issue by smiso** [magento/magento2#16657]
The Get endpoint from catalogProductAttributeManagementV1 of REST API returns Array instead of defined Boolean for options: is_default. I think array would be fine but should also be defined correctly in Swagger definition.

### Preconditions
Magento 2.2.5
Linux
Apache 2.2.31 (Unix) mod_fastcgi
PHP 7.0.30
MySQL 5.6.23-72.1

### Steps to reproduce
Prerequsite: Installation with multiple Store Views and attribute country_of_manufacture with different default options per store view
REST Request on catalogProductAttributeManagementV1 (Get) for attribute set that has attribute country_of_manufacture included

### Expected result
Swagger defines field is_default as boolean:
```
[
  {
    ...
    "options": [
      {
        ...
        "is_default": true,
        ...
```
### Actual result
Magento returns array of store view Id's which has option as default:
```
...
  {
    "attribute_id": 609,
    "attribute_code": "country_of_manufacture",
    "frontend_input": "select",
    "entity_type_id": "4",
    "is_required": false,
    "options": [
      ...
      {
        "label": "Deutschland",
        "value": "DE",
        "is_default": [ "2" ]
      },
      ...
      {
        "label": "Schweden",
        "value": "SE"
      },
      ...
      {
        "label": "Schweiz",
        "value": "CH",
        "is_default": [ "1", "4", "8", "9", "10", "11", "12" ]
      },
...
```

**Comment by magento-engcom-team** [comment]
Hi @smiso. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@smiso do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by engcom-backlog-nickolas** [comment]
Hello @smiso, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by LusienGG** [comment]
Reproduced on 2.2.6

**Comment by magento-engcom-team** [comment]
Hi @pmclain. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by ihor-sviziev** [comment]
Hi @smiso,
According to test results in https://github.com/magento/magento2/pull/18709#issuecomment-581847344 we weren't able to reproduce this issue on 2.4-develop. Could you confirm that it's already fixed?

**Comment by ihor-sviziev** [comment]
Hi @smiso,

Seems like the issue was already fixed in `2.4-develop` branch as we weren't able to reproduce it anymore. Here is test report https://github.com/magento/magento2/pull/18709#issuecomment-581847344
I'm closing this issue.

If you still have it - please report separate issue with steps to reproduce and just link this issue as reference.

**Comment by magento-engcom-team** [comment]
Hi @smiso, @ihor-sviziev. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-24091` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
